shopinvader_locomotive/wizard/change_password.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com).
+# @author Raphaël Reverdy <raphael.reverdy@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ChangePasswordWizard(models.TransientModel):
```

Comment:
The name of the module should be `shopinvader....`
The name of the class should be `ShopinvaderChange...`
The name of the file should be `shopinvader_shange....py`